tests/phpunit/elementor/data/test-manager.php

```diff
@@ -1,31 +1,18 @@
 <?php
-namespace Elementor\Tests\Phpunit\Elementor\Core\Data;
+namespace Elementor\Tests\Phpunit\Elementor\Data;
 
 use Elementor\Data\Base\Processor;
-use Elementor\Data\Manager;
-use Elementor\Testing\Elementor_Test_Base;
-use Elementor\Tests\Phpunit\Elementor\Data\Base\Mock\Template\Controller as ControllerTemplate;
-use Elementor\Tests\Phpunit\Elementor\Data\Base\Mock\Simple\Controller as ControllerSimple;
-use Elementor\Tests\Phpunit\Elementor\Data\Base\Mock\Processor\Controller as ControllerWithProcessor;
+use Elementor\Tests\Phpunit\Elementor\Data\V2\Base\Data_Test_Base;
+use Elementor\Tests\Phpunit\Elementor\Data\V2\Base\Mock\Template\Controller as ControllerTemplate;
+use Elementor\Tests\Phpunit\Elementor\Data\V2\Base\Mock\WithEndpoint\Controller as ControllerWithEndpoint;
+use Elementor\Tests\Phpunit\Elementor\Data\V2\Base\Mock\Processor\Controller as ControllerWithProcessor;
 
 
-class Test_Manager extends Elementor_Test_Base {
-
-	/**
-	 * @var \Elementor\Data\Manager
-	 */
-	protected $manager;
+class Test_Manager extends Data_Test_Base {
 
 	public function setUp() {
 		parent::setUp();
 
-		$this->manager = Manager::instance();
-		$this->manager->kill_server();
-	}
-
-	public function tearDown() {
-		parent::tearDown();
-
 		$this->manager->kill_server();
```

Comment:
Try to move it to the base

Reply:
Cannot it should support backwards compatibility